src/prefect/cli/run.py

```diff
@@ -665,48 +680,73 @@ def run(
             quiet_echo("Aborted.")
         return
 
-    # Exit now if we're not waiting for execution to finish
-    if not watch:
-        return
-
-    try:
-        quiet_echo("Watching flow run execution...")
-        for log in watch_flow_run(
-            flow_run_id=flow_run_id,
-            stream_logs=not no_logs,
-        ):
-            level_name = logging.getLevelName(log.level)
-            timestamp = log.timestamp.in_tz(tz="local")
-            echo_with_log_color(
-                log.level, f"└── {timestamp:%H:%M:%S} | {level_name:<7} | {log.message}"
-            )
-
-    except KeyboardInterrupt:
-        quiet_echo("Keyboard interrupt detected!", fg="yellow")
+    # Handle agentless execution
```

Comment:
The diff in here was very weirdly handled by git -- not much has changed. Might be easier to view with indentation changes ignored?